src/libexpr/value.hh

```diff
@@ -38,6 +39,7 @@ class XMLWriter;
 
 
 typedef long NixInt;
+typedef double NixFloat;
```

Comment:
Thanks for pointing that out -- that's the stuff I'm not readily aware of. :)

If we did that - would that become a problem with Nix behaving differently on 32-bit vs. 64-bit platforms?